src/Engine/EmitMSIL/Operators.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading;
+using ProtoCore.Utils;
+
+namespace EmitMSIL
+{
+    public class Operators
+    {
+        private static string GetOperatorMethodName(string name)
+        {
+            if (name[0] == '%')
+            {
+                name = name.Remove(0, 1);
+            }
+
+            return name[0].ToString().ToUpper() + name.Substring(1);
+        }
+
+        private static bool ToBoolean(dynamic a)
+        {
+            if (a is bool ab)
```

Comment:
just a style thing, I think you can `switch` on type now, though we may not yet use a c# version that supports that.

Reply:
Thanks, I wll give it a try.